magma/bits.py

```diff
@@ -23,6 +23,8 @@
 from magma.language_utils import primitive_to_python
 from magma.logging import root_logger
 from magma.generator import Generator2
+from magma.wire_container import WiringLog
+from magma.debug import debug_wire
```

Comment:
still need these 2? don't see them used

Reply:
debug_wire is being used on line 159, don't need WiringLog anymore though